src/components/PopupWithForm.js

```diff
@@ -19,6 +20,10 @@ export default class PopupWithForm extends Popup {
     return this._formValues;
   }
 
+  handleAvatarChange(link) {
```

Comment:
There should **not** be any code in universal functions that doesn’t suit every popup.